### DISCUSSION TITLE
plotBrowserTrack() results look truncated

### DISCUSSION THREAD

**Discussion by graciegordon** [GreenleafLab/ArchR#936] [Questions / Documentation]
Hello @rcorces and @jgranja24, 

I have a follow up question to discussion [#903](https://github.com/GreenleafLab/ArchR/discussions/903). Is there a way to specify ylims with the signal range rather than the quartile? Using the quartiles with ylims=c(0,1) looks like it might be truncating some of the peaks (ex: NK-B-2). I am using the archR v1.

<img width="405" alt="Screen Shot 2021-07-27 at 12 39 58 PM" src="https://user-images.githubusercontent.com/32146875/127217226-242547ce-4c93-4aac-a69e-ab98114fcc5f.png">
.

**Comment by rcorces** [comment]
I dont have enough information to help you. ArchR "v1" is not specific. As you indicated #903 discusses a bug in the ylim argument. This fix has been implemented in the `release_1.0.2` branch which you can install via:
```devtools::install_github("GreenleafLab/ArchR", ref="master", repos = BiocManager::repositories())```
Then restart your R session and reload ArchR with `library(ArchR)`

Make sure you are using the correct parameter (`ylim` not "ylims") and note that if `ylim` is not set, ArchR defaults to using the `0.999` quantile instead of `1`.
https://github.com/GreenleafLab/ArchR/blob/ef6fd035fe75329aca19d14f3c656503ff3a834e/R/ArchRBrowser.R#L997

It is possible that your "truncating" of the axis is due to:
1) using a version of ArchR that doesnt have the above-mentioned fix implemented
2) incorrect usage of the ylim param
3) Smoothing of the signal that makes it look like the y value is higher than the y axis limit.
